test/jdk/tools/jpackage/share/ErrorTest.java

```diff
@@ -603,7 +630,19 @@ public static Collection<Object[]> testMac() {
                 testSpec().nativeType().addArgs("--mac-app-store", "--runtime-image", Token.JAVA_HOME.token())
                         .error("ERR_MacAppStoreRuntimeBinExists", JPackageCommand.cannedArgument(cmd -> {
                             return Path.of(cmd.getArgumentValue("--runtime-image")).toAbsolutePath();
-                        }, Token.JAVA_HOME.token()))
+                        }, Token.JAVA_HOME.token())),
+                testSpec().noAppDesc().nativeType()
+                        .addArgs("--runtime-image", Token.INVALID_MAC_JDK_BUNDLE.token())
+                        .error("message.runtime-image-invalid", JPackageCommand.cannedArgument(cmd -> {
+                            return Path.of(cmd.getArgumentValue("--runtime-image")).toAbsolutePath();
```

Comment:
Why do we need an absolute path in error messages? Wouldn't it make more sense to have the value of `--runtime-image` parameter as is in error messages?